docs/source/actions.rst

```diff
@@ -182,6 +182,51 @@ In the example above, ``to_number`` parameter contains attribute ``secret``
 which value is ``true``. If an attribute is marked as a secret, value of that
 attribute will be masked in the |st2| service logs.
 
+Parameters in actions
```

Comment:
Yeah, this is not possible now since we don't have DB schema for what goes into context. It's simply a dict which means you can have arbitrary KV pairs. This is both good and bad. I do think though that this is beyond the scope of this PR.